packages/esm-patient-chart-app/src/actions-buttons/start-visit.component.test.tsx

```diff
@@ -9,6 +9,9 @@ const mockUseVisit = useVisit as jest.Mock;
 
 jest.mock('@openmrs/esm-framework', () => ({
   useVisit: jest.fn(),
+  createGlobalStore: jest.fn().mockImplementation(() => {
+    return { action: jest.fn() };
+  }),
```

Comment:
We need to add `useVisit` to https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx so we can get rid of this block altogether.